app/controllers/users_controller.rb

```diff
@@ -27,6 +27,12 @@ def create
     @user = User.new(params.require(:user).permit(:email, :first_name, :last_name, :password, :password_confirmation))
     if @user.save
       session[:user_id] = @user.id
+      @org_invites = OrganizationInvite.where("email = ? and created_at >= ?", @user.email, 1.week.ago)
+      if @org_invites.length > 0
```

Comment:
I believe you can do `if @org_invites.exists?` which will do a SQL command which checks if any exist, which may be more efficient (and it reads a bit better).